### ISSUE TITLE
Olanzapine exposure

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10382]
**Preferred term label:**
Olanzapine exposure

**Synonyms**
Zyprexa

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Can cause acute pancreatitis

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000780
- label: Olanzapine exposure
- parents: [Medication history (HP:4000112)](https://hpo.jax.org/app/browse/term/HP:4000112)
- pmid: PMID:35510088
- def: History of having taken olanzapine.
- comment: Olanzapine exposure has been associated with acute pancreatitis.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10382
- synonyms:
